ui/packages/catalog-ui-search/src/main/webapp/react-component/location/bounding-box.js

```diff
@@ -30,22 +33,46 @@ const { Direction } = require('../../component/location-new/utils/dms-utils.js')
 
 const minimumDifference = 0.0001
 
-const BoundingBoxLatLon = props => {
-  const { north, east, south, west, cursor } = props
-
-  const { mapEast, mapWest, mapSouth, mapNorth } = props
-
+const BoundingBoxLatLonDd = props => {
+  const {
+    north,
+    east,
+    south,
+    west,
+    setState,
+    mapEast,
+    mapWest,
+    mapSouth,
+    mapNorth,
+  } = props
+  const [ddError, setDdError] = useState(initialErrorStateWithDefault)
   const westMax = parseFloat(mapEast) - minimumDifference
   const eastMin = parseFloat(mapWest) + minimumDifference
   const northMin = parseFloat(mapSouth) + minimumDifference
   const southMax = parseFloat(mapNorth) - minimumDifference
 
+  useEffect(() => {
+    if(props.drawing) {
+      setDdError(initialErrorStateWithDefault)
+    }
+  }, [props.east, props.west, props.south, props.north])
+
+  function onChangeDd(key, label, value) {
```

Comment:
for consistency's sake (it's `type` in dms)
```suggestion
  function onChangeDd(key, type, value) {
```